reference/constraints/Timezone.rst

```diff
@@ -0,0 +1,127 @@
+Timezone
+========
+
+Validates that a value is a valid timezone identifier (ie. ``Europe/Paris``).
+
+==========  ===================================================================
+Applies to  :ref:`property or method <validation-property-target>`
+Options     - `groups`_
+            - `message`_
+            - `payload`_
+Class       :class:`Symfony\\Component\\Validator\\Constraints\\Timezone`
+Validator   :class:`Symfony\\Component\\Validator\\Constraints\\TimezoneValidator`
+==========  ===================================================================
+
+Basic Usage
+-----------
+
+Suppose you have a ``UserSettings`` class, with a ``timezone`` field that is a string
+meant to contain a timezone identifier (ie. `America/New_York`):
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        // src/Entity/UserSettings.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class UserSettings
+        {
+            /**
+             * @Assert\Timezone
+             * @var string A timezone identifier
+             */
+             protected $timezone;
+        }
+
+    .. code-block:: yaml
+
+        # config/validator/validation.yaml
+        App\Entity\UserSettings:
+            properties:
+                timezone:
+                    - Timezone: ~
+
+    .. code-block:: xml
+
+        <!-- config/validator/validation.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping https://symfony.com/schema/dic/constraint-mapping/constraint-mapping-1.0.xsd">
+
+            <class name="App\Entity\UserSettings">
+                <property name="timezone">
+                    <constraint name="Timezone"/>
+                </property>
+            </class>
+        </constraint-mapping>
+
+    .. code-block:: php
+
+        // src/Entity/UserSettings.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Mapping\ClassMetadata;
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Event
+        {
+           /**
+            * @var string A timezone identifier
+            */
+            protected $timezone;
+
+            public static function loadValidatorMetadata(ClassMetadata $metadata)
+            {
+                $metadata->addPropertyConstraint('startsAt', new Assert\Timezone());
```

Comment:
Is this a copy & paste error? I would expect it to be `timezone` not `startsAt`?

Reply:
Fixed!